components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -196,10 +196,18 @@ abstract class AbstractFetchDownloadService : Service() {
                                 filePath = currentDownloadJobState.state.filePath,
                                 contentType = currentDownloadJobState.state.contentType
                             )) {
-                            Toast.makeText(applicationContext,
-                                applicationContext.getString(R.string.mozac_feature_downloads_could_not_open_file),
-                                Toast.LENGTH_SHORT
-                            ).show()
+                            val zipExt = ".zip"
+                            if (currentDownloadJobState.state.filePath.endsWith(zipExt)) {
```

Comment:
I thought we were looking to have a generic message where we replace the file extension for all non supported files. Instead of having one specific message for zips. What do you think?

`<string name="mozac_feature_downloads_open_not_supported">Opening %1$s files not supported</string>`


 ```kotlin
val errorMessage = applicationContext.getString(R.string.mozac_feature_downloads_open_not_supported, fileExtension)

Toast.makeText(applicationContext,errorMessage,Toast.LENGTH_SHORT).show()

```

Reply:
I see, I misunderstood! My concern here is that some cases like .zip are not supported.. and potentially others like .rar as well. But, what about the case where we download an image file but for some reason it is corrupted or can't be opened, then we will display a message saying 'Opening PNG not supported'- when we clearly do. I wanted to isolate the cases that are likely caused by Android's own issues with their intent such as .Zip into their own message so that the user can differentiate between some kind of download failure and an inability to support opening files.